torch/_inductor/select_algorithm.py

```diff
@@ -64,6 +64,14 @@ class KernelNamespace:
 extern_kernels = KernelNamespace()
 
 
+@functools.lru_cache(None)
+def warn_ast_parse_3_11():
+    log.warning(
```

Comment:
Do we actually need a warning here? It doesn't matter that much right - only some compilation time improvement? In that case, I think most users wouldn't care.

Reply:
Now that we have a ton more mm configs, it actually matters a good amount.

Reply:
I'd lean towards not having it (i think warning spew is really bad), but if you really want a warning, at least mention what's going to happen if they don't upgrade.

Reply:
Yea, fair. I'll remove and then after this is landed try to add a workaround in triton.